crud/helpers.go

```diff
@@ -162,7 +176,14 @@ func waitForStateRefresh(sync StatefulResource, timeout time.Duration, pending,
 		handleMissingResourceError(sync, &e)
 		return
 	}
-	sync.SetData()
 
 	return
 }
+
+func FilterMissingResourceError(sync ResourceVoider, err *error) {
```

Comment:
ah yes. This was just a public version of `handleMissingResourceError()` for the #15 spike